### PR TITLE
Integrated top consumer card for storage dashboard

### PR DESCRIPTION
**Pull request by gnehapk**
ref: https://github.com/kubevirt/web-ui-components/pull/349

### CODE REVIEW

**Comment by gnehapk** [comment]
@rawagner @mareklibra I have amended the PR. Please review.

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 1]
```diff
@@ -180,6 +201,7 @@ export class StorageOverview extends React.Component {
           },
           ...utilizationData,
           alertsResponse,
+          ...topConsumersData,
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: please do not send such generic props to the consumer. i.e. `stats` and `loaded`

Comment (gnehapk) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: ack

**Comment by gnehapk** [comment]
Incorporated the changes. @rawagner @suomiy @mareklibra Please review.

**Pull request merged**